openedx/core/djangoapps/zendesk_proxy/v1/views.py

```diff
@@ -64,6 +64,9 @@ def post(self, request):
                 custom_fields=request.data['custom_fields'],
                 tags=request.data['tags']
```

Comment:
I can't comment on the actual line (it's too far above ^), but:

The request data allegedly contains the requester's email, so in theory, we could fall back to that value, but I'm averse to sending email to non-verified, user-supplied addresses.

Do we have counter-examples in existence already? Or thoughts in general?

Reply:
^^^ This logic makes sense to me... seems like a problem to use a user-supplied email.

Reply:
Ah, that's actually the change that introduced this;
we switched from pulling email out of the request data and started getting it from the user object, but AnonymousUser doesn't have an email.

- https://github.com/edx/edx-platform/pull/24418

Reply:
Yeah, turns out, this is exactly why the change landed; to stop using client-provided emails.
So we _definitely_ wouldn't want to revert to that ;)